docs/source/coredev/release_process.rst

```diff
@@ -42,20 +49,22 @@ If a major release:
 
           python tools/update_whatsnew.py
 
-    - update `docs/source/whatsnew/development.rst`, to ensure it covers
+    - update ``docs/source/whatsnew/development.rst``, to ensure it covers
       the major release features
-    - move the contents of `development.rst` to `versionX.rst` where `X` is
+
+    - move the contents of ``development.rst`` to ``versionX.rst`` where `X` is
```

Comment:
Does not really matter as our default directive is `pre` anyway. And I understood you ment to type ``` `` `X` `` ```